lib/src/tests/renderOverload.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import {
```

Comment:
I realize those tests are related to the VernaProvider. So I think you could them into the `VernaProvider` test suite.